src/tests/class_tests/openms/source/AASequence_test.cpp

```diff
@@ -232,8 +232,17 @@ START_SECTION((double getAverageWeight(Residue::ResidueType type = Residue::Full
 END_SECTION
 
 START_SECTION((double getMonoWeight(Residue::ResidueType type = Residue::Full, Int charge=0) const))
-  TOLERANCE_ABSOLUTE(1e-6)
-  TOLERANCE_RELATIVE(1.0 + 1e-6)
+  TOLERANCE_ABSOLUTE(1e-5)
+  TOLERANCE_RELATIVE(1.0 + 1e-5)
```

Comment:
That seems like a reasonable solution. @timosachsenberg and I already discussed that part and came up with a similar solution outside of github.
I guess it suffices to test the correctness of each iontype once (according to the compisition) and leave the old tests as they are.